atlascope_plugins/atlascope-vandy-importer/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name="atlascope-vandy-importer",
+    version="0.0.1",
+    entry_points={
+        "atlascope.plugins": "vandy = import_function:run",
+    },
+    install_requires=[
+        "requests",
+        'importlib; python_version == "2.6"',
```

Comment:
Another copy-paste from the `setuptools` quickstart: https://setuptools.pypa.io/en/latest/userguide/quickstart.html#:~:text=setup.cfg-,setup.py,-from%20setuptools%20import

Reply:
Seems odd since we don't even touch Python2 in this project. But it's it's not causing any harm, then we can move along. Might be good to figure out why they have you copypasta'ing this way (but not urgent at all).